README.md

```diff
@@ -1,40 +1,7 @@
-<a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
```

Comment:
I think it would be better to remove all commented out lines in README.md
This helps your work to look good .